### PR TITLE
COMMERCE-2261 fix commerce payment engine test

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

**Comment by marco-leo** [comment]
@riccardo-alberti your CI user is still not configured properly

**Comment by riccardo-alberti** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

**Comment by ethanbustad** [comment]
@riccardo-alberti the failure now is that a version increase is required in `commerce-test-util`, which can be generated with `gw baseline` (keep running this command until it succeeds). You can tell because of the full console:
> For full console, click [here](https://test-1-16.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/144//consoleText).

I always check there if the error message isn't clear.

File: commerce-payment-test/src/testIntegration/java/com/liferay/commerce/payment/engine/test/CommercePaymentEngineTest.java [hunk 1]
```diff
@@ -216,6 +246,18 @@ public void testProcessPayment() throws Exception {
...
+	private CommerceChannel _commerceChannel;
+
+	@Inject
+	private CommerceChannelLocalService _commerceChannelLocalService;
+
+	@DeleteAfterTestRun
+	private CommerceChannelRel _commerceChannelRel;
+
+	@Inject
+	private CommerceChannelRelLocalService _commerceChannelRelServ;
```
Comment (ethanbustad) [inline commerce-payment-test/src/testIntegration/java/com/liferay/commerce/payment/engine/test/CommercePaymentEngineTest.java:259]: Should be named `CommerceChannelRelLocalService _commerceChannelRelLocalService`, for clarity/consistency.

File: commerce-test-util/src/main/java/com/liferay/commerce/test/util/TestCommerceContext.java [hunk 2]
```diff
@@ -95,5 +95,4 @@ public long getSiteGroupId() throws PortalException {
 	private final CommerceOrder _commerceOrder;
 	private final Group _contextGroup;
 	private final User _contextUser;
-
```
Comment (ethanbustad) [inline commerce-test-util/src/main/java/com/liferay/commerce/test/util/TestCommerceContext.java:98]: this empty line is required -- all classes should have an empty line before the closing brace.

File: commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java [hunk 3]
```diff
@@ -45,28 +48,25 @@
...
-
-import java.util.Collections;
-
 /**
  * @author Andrea Di Giorgi
  * @author Luca Pellizzon
  */
 public class CommerceTestUtil {
-
+
```
Comment (ethanbustad) [inline commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java:63]: There's some extra whitespace added in a few places in this file.

File: commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java [hunk 4]
```diff
@@ -103,7 +103,36 @@ public static CPDefinitionInventory addBackOrderCPDefinitionInventory(
...
+			CommerceOrderLocalServiceUtil.updateCommerceOrder(commerceOrder);
+		}
+
+		CommerceContext commerceContext = new TestCommerceContext(
+			commerceOrder.getCommerceCurrency(), 
+			user, 
+			group, 
+			commerceOrder.getCommerceAccount(),
+			commerceOrder);
```
Comment (ethanbustad) [inline commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java:131]: Put as many parameters on a single line as fit in 80 columns. Source formatter should flag anything like this that doesn't conform to standards. (Same with the other whitespace issues in this PR, I think.) So in general, try to be sure to run `gw formatSource` before sending a PR (and you could follow that with `gw checkSourceFormatting` to make sure you got everything).

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by riccardo-alberti** [comment]
ci:test

**Pull request closed without merging**